.github/dependabot.yml

```
@@ -3,15 +3,18 @@ updates:
           - '*'
+    labels:
+      - 'dependencies'
+      - 'automated'
     commit-message:
-      prefix: 'feat'
-      prefix-development: 'chore'
-      include: 'scope'
-      
\ No newline at end of file
+      prefix: 'deps'
+      include: 'scope'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)